pkg/util/util.go

```diff
@@ -14,6 +15,14 @@ func IsURL(str string) bool {
 	return true
 }
 
+func IsLocalPath(str string) bool {
```

Comment:
This concerns me. I think you are working around something wrong with IsURL which is ok. But I’d like to make this and IsURL a single function. And it can be called with ! When you need the other. So can you consolidate these two methods?

Reply:
actually, I ended up not using this... removing